content/07.Publication_Process.md

```diff
@@ -1,15 +1,38 @@
 # The Publication and Peer Review Process 
-
 <!-- Leader: Victoria Nolan; co-leaders: Sprih Harsh --> 
 
+Section outline: This section aims to provide an overview of the research publication and peer review process, both from the perspective of a researcher submitting a manuscript to a journal and from that of the peer reviewer providing comments on the work. This content is aimed at students and early-career researchers who may have no or little knowledge of the publication process, or how to become a successful and good peer-reviewer. It will cover the follow topics:
+
 ## Determining coauthorship 
+- Criteria to determine if a researcher, student, technician etc. qualifies for author contributions 
+- General journal threshold requirements for authorship and how to write a contributions statement
+- Determining the order of the authors
+- Acting as the corresponding author – commitments and duties
 
 ## Choosing the best outlet 
+- How to choose a journal for your work – scope, relevance, audience, etc.
+- Different types of research (article, review, response to article, images, technical notes etc.)
+- Differences between journals (open access, subscription, fees, fee waivers etc.)
+- Importance of journal impact factor and rankings
 
 ## Who should review your manuscript? 
+-	Getting original research proposal reviewed in initial design phase
+-	Asking colleagues/ others in field to review manuscript before submission
+-	When and who to suggest as or to avoid as your assigned editor
+-	How to choose and suggest reviewers if required
 
 ## How to constructively provide feedback as a reviewer? 
```

Comment:
I think it would be good to include a box that shows examples of providing feedback and responding to feedback.